p2p/net/network.go

```diff
@@ -290,27 +293,31 @@ func (n *Net) accept(listen net.Listener) {
 				return
 			}
 			if !Temporary(err) {
-				n.logger.Error("Listener errored while accepting connections: err: %v", err)
+				n.logger.With().Error("listener errored while accepting connections", log.Err(err))
 				return
 			}
 
-			n.logger.Warning("Failed to accept connection request err:%v", err)
+			n.logger.With().Warning("failed to accept connection request", log.Err(err))
 			pending <- struct{}{}
 			continue
 		}
 
-		n.logger.Debug("Got new connection... Remote Address: %s", netConn.RemoteAddr())
+		n.logger.With().Debug("got new connection",
+			log.String("remote_addr", netConn.RemoteAddr().String()),
+			log.String("network", netConn.RemoteAddr().Network()))
 		conn := netConn.(*net.TCPConn)
 		n.tcpSocketConfig(conn) // TODO maybe only set this after session handshake to prevent denial of service with big messages
 		c := newConnection(netConn, n, nil, nil, n.config.MsgSizeLimit, n.config.ResponseTimeout, n.logger)
 		go func(con Connection) {
 			defer func() { pending <- struct{}{} }()
 			err := c.setupIncoming(n.config.SessionTimeout)
 			if err != nil {
-				n.logger.Event().Warning("conn_incoming_failed", log.String("remote", c.remoteAddr.String()), log.Err(err))
+				n.logger.Event().Warning("incoming connection failed",
+					log.String("remote_addr", c.remoteAddr.String()),
+					log.Err(err))
 				return
 			}
-			go c.beginEventProcessing()
+			go c.beginEventProcessing(context.TODO())
```

Comment:
Why not use `context.Background()`? Do you think we should pass a context from the outside here in the future?

Reply:
you're right, this one is not a TODO. fixed.